easy-ui-react/src/RadioButtonGroup/RadioButtonGroup.module.scss

```
@@ -0,0 +1,47 @@
+@use "../styles/common" as *;
+@use "../Button/mixins" as Button;
+@use "../styles/unstyled";
+
+.RadioButtonGroupButton {
+  @include unstyled.button;
+  min-width: 32px;
+  padding: calc(#{design-token("space.0.5")} - 1px) // subracting 1 for border
+    design-token("space.1");
+  border: design-token("shape.border_width.1") solid
+    design-token("color.neutral.300");
+  outline: none;
+  color: design-token("color.primary.800");
+  margin-inline-start: calc(#{design-token("shape.border_width.1")} * -1);
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+
+  &:first-of-type {
```

```suggestion
  &:first-child {
```

`:first-of-type` can be a bit finicky. i would just use `:first-child` and `:last-child`